player/src/main/java/com/focusit/jsflight/player/webdriver/SeleniumDriver.java

```diff
@@ -56,25 +68,49 @@ public SeleniumDriver(UserScenario scenario, Map<String, String> externalLastUrl
     {
         this.scenario = scenario;
         this.lastUrls = externalLastUrls;
+        //This must be set due to equals of WebElement
+        NO_OP_ELEMENT.setId("NO_OP");
+    }
+
+    public void setUseRandomChars(boolean useRandomChars)
```

Comment:
it might be more elegant if setter like this one will return "this". So you can chain them. just saying